logic-apps/univariate-anomaly-detection-last-point.json

```diff
@@ -0,0 +1,247 @@
+{
+  "definition": {
+    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+    "actions": {
+      "Check_If_Last_Point_Is_Anomaly": {
+        "inputs": {
+          "authentication": "@parameters('AnomalyDetectorIdentityAuthentication')",
+          "body": "@body('Parse_Content_of_the_Message')",
+          "headers": {
+            "Content-Type": "application/json"
+          },
+          "method": "POST",
+          "uri": "@{parameters('AnomalyDetectionEndpoint')}anomalydetector/v1.1/timeseries/last/detect"
+        },
+        "runAfter": {
+          "Parse_Content_of_the_Message": [
+            "Succeeded"
+          ]
+        },
+        "type": "Http"
+      },
+      "Complete_the_message_in_a_queue": {
+        "inputs": {
+          "host": {
+            "connection": {
+              "name": "@parameters('$connections')['servicebus']['connectionId']"
+            }
+          },
+          "method": "delete",
+          "path": "/@{encodeURIComponent(encodeURIComponent('anomaly-detection-queue'))}/messages/complete",
+          "queries": {
+            "lockToken": "@triggerBody()?['LockToken']",
+            "queueType": "Main"
+          }
+        },
+        "runAfter": {
+          "If_Status_Code_Is_200": [
+            "Succeeded"
+          ]
+        },
+        "type": "ApiConnection"
+      },
+      "If_Status_Code_Is_200": {
```

Comment:
The maximum count of peek-lock action in SB is 10 https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues#maximum-delivery-count . Currently, in any scenario of the logic app it will delete the message from the queue.